### PR TITLE
Issue #13345: Enable examples tests for IllegalImportCheckExamplesTest

### PR DESCRIPTION
**Pull request by luvtyping**
### Enable examples tests for IllegalImportCheckExamplesTest
Issue #13345

### CODE REVIEW

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example1.java [hunk 1]
```diff
@@ -6,5 +6,9 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.imports.illegalimport;
+
 // xdoc section -- start
 // xdoc section -- end
+
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example1.java:13]: Please copy imports from other example classes.
Please pay attention to issue description images that shows how examples should be different from each other. It is ok to change them , they should be same except for trailing comments.

**Review by romani — CHANGES_REQUESTED** [review]
Items to improve

**Comment by romani** [comment]
We need to put example.java before `To configure the check so that it rejects packages java.io.* and java.sql.*` phrase, please update template file to insert there content of Example1.java.

You can build Web site on local `mvn clean  site -Pno-validations` website will be in target folder.

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example4.java [hunk 2]
```diff
@@ -17,7 +19,6 @@
 import java.util.Enumeration;   // violation
 import java.util.Arrays;        // violation
 import java.util.Date;          // violation
-import sun.applet.*;
```
[the remark this answers is not in the document]
Comment (romani) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example4.java:20]: Please move trailing comments closer to `;`

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example1.java [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
 // xdoc section -- start
 import java.io.*;
 import java.lang.ArithmeticException;
+import java.sql.Connection;
 import java.util.List;
 import java.util.Enumeration;
 import java.util.Arrays;
-import sun.applet.*;
+import java.util.Date;
```
[the remark this answers is not in the document]
Comment (luvtyping) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example1.java:18]: Should I restore `sun.` for all examples or just Example1?

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example1.java (hunk 1 above)
Comment (luvtyping) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example1.java]: done

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example1.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (luvtyping) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example1.java]: done

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example4.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (luvtyping) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example4.java]: done

**Comment by luvtyping** [comment]
GitHub, generate website

File: src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example1.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+// xdoc section -- start
+import java.io.*;
+import java.lang.ArithmeticException;
+import java.sql.Connection;
+import java.util.List;
+import java.util.Enumeration;
+import java.util.Arrays;
+import java.util.Date;
+import sun.applet.*; // violation, illegal import of all sun packages by default
```
Comment (romani) [inline src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example1.java:20]: Please put in single quotes real violation part.

Example https://github.com/checkstyle/checkstyle/commit/9db7149daa9466798d8915f70527de1780be06ec#diff-c8eb9255bd0ecac76e703a44339630ceabf20d576c9a6db4985bd204fbd6072dR16

Please same for all lines

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example1.java (hunk 4 above)
Comment (luvtyping) [reply src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/Example1.java]: done

**Comment by romani** [comment]
GitHub, generate website

**Review by romani — APPROVED** [review]
Ok to merge thanks a lot.

Please be welcome with more contributions

**Pull request merged**